pages/projects/vest.tsx

```diff
@@ -0,0 +1,16 @@
+import Navbar from '../../components/layout/navbar/navbar';
+import style from '../../styles/projects.module.scss';
+
+export default function vast() {
+  return (
+    <div>
+      <Navbar />
+      <div className={style.container}>
+        <h3 className={style.title}>Vest</h3>
+        <div className={style.star}>
```

Comment:
please use before/ after (on the number of stars element) to create the star element